src/audio/audio.cpp

```diff
@@ -368,6 +368,35 @@ void Audio::playGroupAudioQueued(void*,int group, int peer, const int16_t* data,
     emit static_cast<Core*>(core)->groupPeerAudioPlaying(group, peer);
 }
 
+/**
+Must be called from the audio thread, plays a group call's received audio
+*/
+void Audio::playGroupAudio(int group, int peer, const int16_t* data,
+                           unsigned samples, uint8_t channels, unsigned sample_rate)
+{
+    assert(QThread::currentThread() == audioThread);
+    QMutexLocker locker(&audioLock);
+
+    if (!CoreAV::groupCalls.contains(group))
+        return;
+
+    ToxGroupCall& call = CoreAV::groupCalls[group];
+
+    if (call.inactive || call.muteVol)
```

Comment:
I'll copy the `call.inactive` into #3183, thus it does the same thing - just the right way. :)